src/jit/gentree.cpp

```diff
@@ -7817,6 +7817,504 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     }
 }
 
+GenTreeOperandIterator::GenTreeOperandIterator()
+    : m_node(nullptr)
+    , m_operand(nullptr)
+    , m_argList(nullptr)
+    , m_multiRegArg(nullptr)
+    , m_expandMultiRegArgs(false)
+    , m_state(-1)
+{
+}
+
```

Comment:
`GTF_REVERSE_OPS` is important for updating liveness information at codegen time. This is needed minimally for correct GC info. The reason it matters is that we don't update the liveness of register candidate lclVars at the time we encounter them in the linear sequence. Instead, we update their liveness at the time they are used. If there are two operands that are the same lclVar, we have to update them in order, otherwise we have a last use followed by a non-last-use, which is an error.